Source/MTBBlockedMessage.m

```diff
@@ -278,6 +281,7 @@ - (NSDictionary*)getTrackerDict {
             @"nethunt.com/api/v1/track/email/",
             @"nethunt.co(.*)\\?/pixel.gif"
         ],
+        @"Newsletter2Go": @[@"amxe.net/\\S+.gif"],
```

Comment:
Just noticed this is a duplicate of `SendInBlue`. Probably would make sense to remove this one since they're defunct now right?

Reply:
Yeah, right. I wanted to remove this line when moving it to l. 339 but forgot...
So makes sense to remove it, yes :)